live-usb-creator/README.md

```diff
@@ -28,32 +28,53 @@ This workflow uses the tool [Vagrant](https://www.vagrantup.com/) to orchestrate
-# remove the existing centos/7 image, if one exists
-vagrant box remove -f centos/7
-# install centos 7 vagrant box version 1905.1 (checksum via https://cloud.centos.org/centos/7/vagrant/x86_64/images/sha256sum.txt.asc)
-vagrant box add --checksum-type sha256 --checksum de768cf0180d712a6eac1944bd78870c060999a8b6617f8f9af98ddbb9f2d271 --provider virtualbox --box-version 1905.1 centos/7
-# boot up a virtualbox VM to run the install in
-vagrant up
-# enter the VM
-vagrant ssh
-# kick off the build, from the VM (takes about 25 minutes)
-/vagrant/build.sh
-# will create boot.iso (1.1G) in the current directory (i.e. outside the VM) when finished
-```
+* Install the toolchain:
+
+  ```bash
+  # install virtualbox using brew cask if virtualbox is not installed
+  brew cask install virtualbox
+  # install vagrant using brew cask if vagrant is not installed
+  brew cask install vagrant
+  # install vagrant-vbguest if vagrant-vbguest is not installed
+  vagrant plugin install vagrant-vbguest
+  ```
+
+* Build the ISO:
+
+  ```bash
+  # remove the existing centos/7 image, if one exists
+  vagrant box remove -f centos/7
+  # install centos 7 vagrant box version 1905.1 (checksum via https://cloud.centos.org/centos/7/vagrant/x86_64/images/sha256sum.txt.asc)
+  vagrant box add --checksum-type sha256 --checksum de768cf0180d712a6eac1944bd78870c060999a8b6617f8f9af98ddbb9f2d271 --provider virtualbox --box-version 1905.1 centos/7
+  # boot up a virtualbox VM to run the install in
+  vagrant up
+  # enter the VM
+  vagrant ssh
+  # kick off the build, from the VM (takes about 5-25 minutes), to build a release image. For building a development image, use `/vagrant/build.sh dev` instead
+  /vagrant/build.sh
+  # will create boot.iso (1.1G) in the current directory (i.e. outside the VM) when finished
+  ```
+
+* Rebuid the ISO:
+
+  ```bash
+  # Logout from the VM
+  exit
+  # Halt the VM
+  vagrant halt
+  # Remove the existing centos/7 image
+  vagrant box remove -f centos/7
+  # Clean up the git repo, preserving the artifacts obtained out-of-band (gitignored)
+  git clean -fd
+  # Following the steps in "Build the ISO" above to rebuild the ISO
+  ```
 
 ## Writing image to USB drive
 
 1. Identify device path to disk (such as `/dev/disk3`) with `diskutil list external physical`
 2. Unmount all volumes on that disk with `diskutil unmountDisk /dev/disk3`
 3. Identify the “raw” device path by replacing `disk` with `rdisk` in the device path. (E.g. `/dev/disk3` becomes `/dev/rdisk3`.) This speeds up writes by 3–4× in very informal testing.
-3. Write ISO to that “raw” device path with `sudo dd bs=1m if=boot.iso of=/dev/rdisk5`. Remember that on MacOS you can check in on progress by sending SIGINFO with Ctrl+T. (Takes about 30 minutes in one test.)
+3. Write ISO to that “raw” device path with `sudo dd bs=1m if=boot.iso of=/dev/rdisk3`. Remember that on MacOS you can check in on progress by sending SIGINFO with Ctrl+T. (Takes about 3 minutes in one test.)
```

Comment:
I would put this as `/dev/rdiskN`, since the N is going to be different for everyone.

Reply:
👍  Good idea. (I was intending to keep the changes to README small with this PR, and `disk3` is a more consistent name in the context. But I think it's good to be more rigorous here.)